tests/e2e/cairo1/test_cairo1_testing.py

```diff
@@ -51,8 +51,12 @@ def test_failing_tests(protostar: ProtostarFixture, copy_fixture: CopyFixture):
         "Test failed with data: ",
         "[1870930782904301745253, 482670963043, 31066316372818838395891839589] (integer representation)",
         "['elaborate', 'panic', 'data is here'] (short-string representation)",
+        "[FAIL] tests/failing_test.cairo test_assert_err_code_not_ascii",
+        "Test failed with data:",
+        "[1234] (integer representation)",
+        "[None] (short-string representation)",
```

Comment:
Shouldn't non decodable values be presented as `""` or something like that instead of `None`?

Reply:
I guess we can keep it as `None` then.